analytics_dashboard/static/sass/style-application.scss

```diff
@@ -7,7 +7,7 @@
 // config
 // --------------------
 @import "config-assets";
-@import "config-variables";
+@import "themes/credo/config-variables";
```

Comment:
Instead of duplicating these files and making our changes, I'd suggest we continue to import the edX config-variables, base, and developer files and then also add a `@import 'theme';` to pull in our further customizations.  Otherwise maintaining changes in our copy of these stylesheets is going to be pretty painful.

How does the LMS do the conditional import of an optional theme sass if it exists?  Ideally we'd mimic that.